Documentation/ceph-cluster-crd.md

```diff
@@ -236,7 +236,7 @@ The following are the settings for Storage Class Device Sets which can be config
 * `portable`: If `true`, the OSDs will be allowed to move between nodes during failover. This requires a storage class that supports portability (e.g. `aws-ebs`, but not the local storage provisioner). If `false`, the OSDs will be assigned to a node permanently. Rook will configure Ceph's CRUSH map to support the portability.
 * `volumeClaimTemplates`: A list of PVC templates to use for provisioning the underlying storage devices.
   * `resources.requests.storage`: The desired capacity for the underlying storage devices.
-  * `storageClassName`: The StorageClass to provision PVCs from. Default would be to use the cluster-default StorageClass.
+  * `storageClassName`: The StorageClass to provision PVCs from. Default would be to use the cluster-default StorageClass. This StorageClass should provide raw block device, partition, encrypted device, or logical volume.
```

Comment:
How about if you also add an example of how to use an LV in this scenario? Or it just depends on the storage class having those options? How does the OSD provisioning code know which type it needs to provision?

Reply:
This PR looks like it just enables LVs. This comment also indicates partitions and encrypted devices. How were they enabled?

Reply:
> Or it just depends on the storage class having those options?

Yes, it does.

> How does the OSD provisioning code know which type it needs to provision?

It's not for OSD provisioning code, but for users who specify the storage class. I added the list of device types to let users know which type of PV (block device) is accepted by this feature.

> This PR looks like it just enables LVs. This comment also indicates partitions and encrypted devices. How were they enabled?

It doesn't mean what I added to support (LV), but means the supported device types described in the following line. 

https://github.com/rook/rook/blob/6d8f66cba37e86d3168c8586edfbda129016d816/pkg/clusterd/disk.go#L83

If this line is not necessary, I'll remove it or use a more proper expression.

Reply:
This PR only adds support for LVs. Should the statement be the following? 
```
This StorageClass should provide a raw block device or logical volume. Other types are not supported.
```
We're not adding support for partitions or encrypted devices, right?

Reply:
You're right, will fix.